Add InfluxDB service

sdil opened the pull request.

ripienaar commented:
also seems to be a bit out of date, if you can pls make against latest master that'd be great - you can do both in the same PR if you want

sdil commented:
> also seems to be a bit out of date, if you can pls make against latest master that'd be great - you can do both in the same PR if you want

Fixed. Updated to latest code :)
